newIDE/app/public/external/yarn/README.md

```diff
@@ -0,0 +1,7 @@
+This folder contains sources to embed Yarn editor (https://github.com/InfiniteAmmoInc/Yarn) so that it can
+be used directly from GDevelop to edit/create Dialogue Trees.
+
+Yarn sources are downloaded by `import-zipped-editor.js` script. They are raw, unchanged sources
```

Comment:
Thanks for the README :+1:

Reply:
no probs, just following the convention established with the other ones :)